packages/mask/src/plugins/Pets/SNSAdaptor/ModelView.tsx

```diff
@@ -0,0 +1,35 @@
+import type { CSSProperties } from 'react'
+import '@webcomponents/custom-elements'
+import '@google/model-viewer/dist/model-viewer'
+import { Punk3D } from '../constants'
+
+declare global {
+    namespace JSX {
+        interface IntrinsicElements {
+            'model-viewer': any
+        }
+    }
+}
+
+interface ModelViewProps {
+    styleContent?: CSSProperties | undefined
+    source: string
+}
+
+const ModelView = (props: ModelViewProps) => {
+    const { styleContent, source } = props
+    return (
+        <div style={styleContent}>
+            <model-viewer
```

Comment:
Let `ModelView` inherit `div`, then you can use it like a `div`

```ts
interface ModelViewProps extends HTMLProps<HTMLDivElement> {
    source: string
}
```
Don't forget to update the usages as well.

```tsx
<ModelView style={style} source={source} />
```
```suggestion
import type { FC, HTMLProps } from 'react'
import '@webcomponents/custom-elements'
import '@google/model-viewer/dist/model-viewer'
import { Punk3D } from '../constants'

interface ModelViewElementProps extends HTMLProps<HTMLDivElement> {
    'shadow-intensity': string
    'camera-controls': boolean
    'auto-rotate': boolean
    ar: boolean
}

declare global {
    namespace JSX {
        interface IntrinsicElements {
            'model-viewer': ModelViewElementProps
        }
    }
}

interface ModelViewProps extends HTMLProps<HTMLDivElement> {
    source: string
}

const ModelView: FC<ModelViewProps> = ({ source, ...rest }) => {
    return (
        <div {...rest}>
            <model-viewer
```